config/initializers/global_functions.rb

```diff
@@ -10,10 +10,18 @@ def production?
   Rails.env.production?
 end
 
+def staging?
+  Rails.env.production?
```

Comment:
wtf? that's one deceiving name if I ever saw one :)

production != staging, they're different environments